PFOffer.sol

```diff
@@ -263,7 +263,7 @@ contract PFOffer {
     // The proposal will not accept the results of the vote if it wasn't able
     // to be sure that YEA was able to succeed 48 hours before the deadline
     function checkVoteStatus() noEther {
-        var (,,,votingDeadline,,,,,,yea,nay,) = client.proposals(proposalID);
+        var (,,,votingDeadline,,,,,,,yea,nay,) = client.proposals(proposalID);
```

Comment:
@CJentzsch  This will not work. There is a [solidity bug](https://github.com/ethereum/solidity/issues/598) that is [solved](https://github.com/ethereum/solidity/pull/624) in develop and will hopefully soon make it in a release. Just don't touch PFOffer.sol for this PR.